internal/kconfig/kconfig_test.go

```diff
@@ -39,21 +59,41 @@ func TestParseGziped(t *testing.T) {
 	}
 	defer f.Close()
 
-	_, err = Parse(f)
+	_, err = Parse(f, nil)
+	if err != nil {
+		t.Fatal("Error parsing gziped kconfig: ", err)
+	}
+}
+
+func TestParseGzipedFiltered(t *testing.T) {
```

Comment:
Seems like you removed the simple test case that didn't use the fedora kconfig? I liked that one since it's simple and asserts that we get the right values out. We're missing that coverage now?

I'd prefer it if we had two tests:
* TestParse against a minimal kconfig which asserts values
* TestParseFiltered against a minimal kconfig which asserts the presence of keys and size of map